pom.xml

```
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.19</version>
+    <version>4.78</version>
```

Better to rely on the most recent release of the plugin parent pom rather than relying on an outdated but modern version
```suggestion
    <version>4.85</version>
```